docs/content/testnet/rvasps.en.md

```diff
@@ -101,26 +103,76 @@ To use the protocol buffers directly, use the `TRISAIntegration` service `Transf
 
 These values have the same exact specification as the ones in the command line program.
 
-### Sending a TRISA message to an rVASP
+#### Responding to a TRISA message from an rVASP
+
+Envelopes received from rVASPs can be decrypted using the private unsealing key paired with the public key you exchanged with the rVASP. The payload that you receive from the rVASP will be determined by the originator policy associated with the originator address in the transaction payload.
+
+It is then up to your TRISA node to determine how to handle the payload. Your options are:
+
+1. Reject: return an error in the secure envelope without a payload
+2. Pending: return a `trisa.data.generic.v1beta1.Pending` in the transaction payload
+3. Accept: ensure the identity payload is complete and return payload with `received_at`
+
+The rVASP will handles each type of response appropriately. If a reject message is returned, the rVASP fails the transaction; if accept it "executes" the transaction; and if pending, it puts the transaction into an "await" state.
+
+#### Originator Policies
 
-The rVASP expects a `trisa.data.generic.v1beta1.Transaction` as the transaction payload and an `ivms101.IdentityPayload` as the identity payload. The identity payload beneficiary information does not need to be populated, the rVASP will respond populating the beneficiary, however the identity payload should not be null. It is recommended that you specify some fake identity data to take advantage of the rVASP parsing and validation commands.
+An _originator policy_ determines how the rVASP constructs an outgoing payload and handles the response from the transfer. The originator policy is loaded when the `rvasp` command is used to instruct the rVASP to send an outgoing message either to another rVASP or to your own TRISA node. Originator policies are mapped to wallet addresses, and the rVASP identifies the originator address using the wallet address in the transaction payload. A description of the originator policies follow.
 
-Make sure that in your transaction payload you specify a beneficiary wallet that matches the rVASP beneficiaries from the table above; e.g. use:
+##### SendPartial
+
+For the `SendPartial` policy, the rVASP sends an envelope containing a complete transaction payload and a partial identity payload containing a full originator identity and a partial beneficiary identity. It expects a response envelope containing the full beneficiary information and a `received_at` timestamp in order to complete the transaction.
```

Comment:
What happens if the rVASP gets a `Pending` message back? It shouldn't require the `received_at` timestamp at that point. 

Also we should chat about "complete transaction payload" vs "partial transaction payload" - compliance information transfers are supposed to happen before the transaction, which means that the transaction doesn't have complete information until the very end in the normal case. Over slack, I've sent the documentation I've done for Qredo that might help us discuss what we should document for the rVASP and in the TRISA docs in general.